qiskit/providers/basebackend.py

```diff
@@ -62,6 +62,30 @@ def configuration(self):
         """
         return self._configuration
 
+    def faulty_qubits(self):
```

Comment:
this should not be a method of base backend. it should be in `qiskit/providers/models/backendproperties.py`

Reply:
Moved in db52ca406